hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -369,6 +379,12 @@ tooltip.target.status.pending = Status pending
 tooltip.target.status.registered = Status registered
 tooltip.target.pin = Pin target
 tooltip.distribution.set.pin = Pin distribution set
+tooltip.invalidate.distributionset = Invalidate distribution set
+tooltip.distributionset.invalid = invalid
+tooltip.distributionset.incomplete = incomplete
+tooltip.distributionset.invalidate.forced = Cancelation of open actions will be forced
```

Comment:
Maybe phrase it like "Open actions will be canceled and force quit" as we also use "force quit" in the action history grid

Reply:
Done - changed the tooltip string as suggested